sdk/storage/inc/datalake/datalake_file_system_client.hpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/storage_credential.hpp"
+#include "common/storage_url_builder.hpp"
+#include "datalake_options.hpp"
+#include "http/pipeline.hpp"
+#include "protocol/datalake_rest_client.hpp"
+#include "datalake/datalake_service_client.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  class DataLakePathClient;
+
+  class DataLakeFileSystemClient {
```

Comment:
Do we have to repeat DataLake here? it's already in Datalake namespace.

Reply:
ditto for **every** function names and variable names

Reply:
Removed `DataLake` prefix.